promiseitwonthurt.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+const workshopper = require('workshopper')
+    , path        = require('path')
+
+function fpath (f) {
+  return path.join(__dirname, f)
+}
+
+workshopper({
+    name        : 'promise-it-wont-hurt'
+  , appDir      : __dirname
+  , languages   : ['en', 'fr']
```

Comment:
Does workshopper handle English automatically as it doesn't have the `_en` suffix?